src/app/donation-complete/donation-complete.component.html

```diff
@@ -14,115 +17,97 @@
-          </div>
-          <div class="c-misc">
-            <p>Boost your impact by sharing on social media</p>
-            <a href="https://www.facebook.com/sharer/sharer.php?u={{ shareUrl }}" target="_blank" rel="noopener"><img class="c-social-image" src="/assets/images/share-facebook.png" alt="Facebook" aria-label="Share on Facebook" width="50"></a>
-            <a href="https://twitter.com/share?url={{ shareUrl }}&text={{ prefilledText }}" target="_blank" rel="noopener"><img class="c-social-image" src="/assets/images/share-twitter.png" alt="Twitter" aria-label="Share on Twitter" width="50"></a>
-            <a href="https://www.linkedin.com/cws/share?url={{ shareUrl }}" target="_blank" rel="noopener"><img class="c-social-image" src="/assets/images/share-linkedin.png" alt="LinkedIn" aria-label="Share on LinkedIn" width="50"></a>
-            <a href="https://api.whatsapp.com/send?text={{ prefilledText }}%0A%0A{{ shareUrl }}" target="_blank" rel="noopener"><img id="whatsapp" class="c-social-image" src="/assets/images/share-whatsapp.png" alt="WhatsApp" aria-label="Share on Whatsapp" width="50"></a>
-          </div>
+    <div class="thank-you">
+      <h2>Thank you!</h2>
+
+      <p class="thank-you-text">Your generous donation has<br/>been made successfully</p>
+
+      <div id="save-my-details-cta" *ngIf="!registrationComplete" >
+        <div class="button-wrapper">
+          <biggive-button
+            space-below="0"
+            colour-scheme="white"
+            is-past-campaign="false"
+            is-future-campaign="false"
+            label="Save my details"
+            open-in-new-tab="false"
+            full-width="false"
+            size="medium"
+            rounded="false"
+            centered="false"
+            (click)="openSetPasswordDialog()">
+          </biggive-button>
         </div>
+        <p>This helps you to make doubling your donation easier.</p>
       </div>
 
-      <button class="c-button b-w-100 b-center" mat-button>
-        <div *ngIf="campaign">
-          <!-- For now, this assumes parent / meta-campaigns all have slugs set. -->
-          <p class="b-rt-1 b-m-0" *ngIf="campaign.parentRef"><a [routerLink]="'/' + campaign.parentRef">Explore other campaigns</a></p>
-          <p class="b-rt-1 b-m-0" *ngIf="!campaign.parentRef"><a routerLink="/explore">Explore other campaigns</a></p>
-        </div>
-        <a routerLink="/"><img class="c-button__image" src="/assets/images/logo.png" alt="The Big Give" width="200"></a>
-      </button>
+      <p *ngIf="registrationComplete" id="registrationCompleteText">
+        <strong>You are now registered<span *ngIf="loggedIn"> and logged in</span>.</strong> You can now donate faster using your chosen email address and password.
+      </p>
+
+      <p *ngIf="registerError" class="error" aria-live="assertive">
+        Your password could not be set. Please ensure you chose one with at least 10 characters.
```

Comment:
Probably worth following [ThreeStrikesAndYouRefactor](https://wiki.c2.com/?ThreeStrikesAndYouRefactor) and DRYing up the number 10? It's also used in donation-complete-set-password-dialog.html and reset-password.component.html

Reply:
Looking again I see you're not introducing the third usage of the 10 in this PR - it's just moved from further up the file. So feel free to disregard this if you want.

Reply:
Fixed